packages/opentelemetry-tracing/test/MultiSpanProcessor.test.ts

```diff
@@ -91,15 +92,16 @@ describe('MultiSpanProcessor', () => {
     assert.strictEqual(processor1.spans.length, processor2.spans.length);
 
     span.end();
+    await new Promise(resolve => setTimeout(resolve));
     assert.strictEqual(processor1.spans.length, 1);
-    assert.strictEqual(processor1.spans.length, processor2.spans.length);
+    assert.strictEqual(processor2.spans.length, 1);
 
     await multiSpanProcessor.shutdown();
     assert.strictEqual(processor1.spans.length, 0);
-    assert.strictEqual(processor1.spans.length, processor2.spans.length);
+    assert.strictEqual(processor2.spans.length, 0);
   });
 
-  it('should export spans on graceful shutdown from two span processor', () => {
+  it('should export spans on graceful shutdown from two span processors', done => {
```

Comment:
@obecny this test fails in browser because the unload event handler is actually never called. It wasn't called previously, but it was never caught because the assertions are never run. I changed the test to have a `done` callback and wait for finish, and it never finishes.